tests/acceptance/pageObjects/filesPage.js

```diff
@@ -315,6 +315,46 @@ module.exports = {
 
       // Execute copy
       return client.page.locationPicker().selectFolderAndConfirm(target)
+    },
+
+    /**
+     * Create a md file with the given name
+     *
+     * @param {string} name to set or null to use default value from dialog
+     * @param {boolean} expectToSucceed
+     */
+    createMarkdownFile: async function(name, expectToSucceed = true) {
+      await this.waitForElementVisible('@newFileMenuButton')
+        .click('@newFileMenuButton')
+        .waitForElementVisible('@newMdFileButton')
+        .click('@newMdFileButton')
+        .waitForElementVisible('@dialog')
+        .waitForAnimationToFinish()
+
+      if (name !== null) {
+        await this.clearValueWithEvent('@dialogInput')
+        await this.setValue('@dialogInput', name)
+      }
```

Comment:
name can be `undefined` too especially if nothing is passed to the function. It will not work in that case I think it will break.
Change it to something like this
```suggestion
      if (!name) {
        await this.clearValueWithEvent('@dialogInput')
        await this.setValue('@dialogInput', name)
      }
```